lib/site_settings/deprecated_settings.rb

```diff
@@ -33,6 +33,7 @@ module SiteSettings::DeprecatedSettings
     ],
     ["min_trust_level_to_allow_invite", "invite_allowed_groups", false, "3.3"],
     ["min_trust_level_to_allow_ignore", "ignore_allowed_groups", false, "3.3"],
+    ["min_trust_to_create_tag", "create_tag_allowed_groups", false, "3.3"],
```

Comment:
I wonder if there is a way for us to map the setting of `min_trust_to_create_tag` to `create_tag_allowed_groups` so that we don't have to keep `min_trust_to_create_tag` around and have to check two site settings whenever we want to check if a user can create a tag.

Reply:
No there's not; we are happy with the process we are following here. Certain settings will have to be both checked until all the plugins are all migrated across, we have done this many times now and it's only temporary.

Reply:
That doesn't seem to be safe. If you allow writes to both site settings, how are we going to reconcile it when it is time to drop the old site setting?